tslint.json

```diff
@@ -30,7 +30,7 @@
     "no-eval": true,
     "no-string-literal": false,
     "no-switch-case-fall-through": false,
-    "no-trailing-comma": true,
+    "trailing-comma": [false],
```

Comment:
Can we change this to force trailing comma's? it's more in line with the airbnb standard we try to follow.